src/core/Global.ttslua

```diff
@@ -983,6 +983,15 @@ function removeHelperObject(name)
   end
 end
 
+-- loads saved options
+function loadSettings(newOptions)
+  optionPanel = newOptions
+  updateOptionPanelState()
+  for id, state in pairs(optionPanel) do
+    applyOptionPanelChange(id, state)
+  end
+end
+
```

Comment:
Not sure what the cause is, but in my testing the handhelper and search assistant were spawned despite the option not being ticked in the original save.

Reply:
Yeah it was a some weirdness due to the fact those two are tracked as tables rather than booleans in the `optionPanel` table, so I added a simple check for it and it seems to be golden now.